src/components/DataTypes/Object.tsx

```diff
@@ -65,7 +65,7 @@ export const PreObjectType: React.FC<DataItemProps<object>> = (props) => {
             <CircularArrowsIcon sx={{
               fontSize: 12,
               color: textColor,
-              mx: sizeOfValue ? 0.5 : 0
+              mx: 1
```

Comment:
does this cause the icon too wide? I'm not a UI designer not sure which is better

![Screenshot from 2022-09-22 19-37-20](https://user-images.githubusercontent.com/14026360/191873759-938b7670-4178-40c9-af53-517b7ea33ab7.png)

Reply:
Decreased to `0.5` that means 4px on each side (I hope :))

Reply:
Yeap. That's better.